pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 python = "^3.12"
 datasets = "*"
 pandas = "*"
-litellm = "^1.54.1"
+litellm = "^1.55.4"
```

Comment:
There's an issue with passing `response_format` parameters (used by the `LLMAttentionCondenser`) when using a LiteLLM proxy that was fixed in v. 1.55.4 (see here: https://github.com/BerriAI/litellm/issues/7396).

Reply:
Gotcha, this is fine, just curious though: in the lock file it's 1.55.12, so it shouldn't have mattered?

Just a side question too: do we need `response_format`?

Reply:
Before I bumped it up it was pinned at `1.55.3` in the lock file.

Not sure if `response_format` is _needed_, but it does make getting structural information from an LM much easier and means the `condense` function of `LLMAttentionCondenser` doesn't have to worry about parsing/retry/etc.

Reply:
Yes, sorry, I just saw it after I made this comment. Sure, looks good.

I don't know how well is this parameter supported or how, in various other LLMs than Claude, but we don't need to worry about it in this PR. I assume you have used Claude Sonnet?

Reply:
[This page](https://docs.litellm.ai/docs/completion/json_mode) would indicate pretty broad support. I've added some validation on the relevant condenser to only accept supported LLM configurations.